x/wasm/internal/types/types.go

```diff
@@ -1,13 +1,17 @@
 package types
 
 import (
-	wasmTypes "github.com/CosmWasm/go-cosmwasm/types"
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-const defaultLRUCacheSize = uint64(0)
-const defaultQueryGasLimit = uint64(3000000)
+const (
+	DisabledLRUCacheSize     uint32 = 0
```

Comment:
Is this constant needed? I don't think we need a special case for disabling the cache. Just set to 0 if you don't want to spend momre than 0 for the cache.